src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { TodoInfo } from '../TodoInfo';
+
+interface Props {
+  todos: Todo[];
+  users: User[];
```

Comment:
The `users` prop is defined in the `Props` interface but is not used in the `TodoList` component. Consider removing this prop if it is not needed.

Reply:
TS gives an error without this type in props: users={users} - Type '{ todos: Todo[]; users: User[]; }' is not assignable to type 'IntrinsicAttributes & Props'.
  Property 'users' does not exist on type 'IntrinsicAttributes & Props'.ts(2322)
(property) users: User[]